pkg/config/config.go

```diff
@@ -192,6 +194,7 @@ func DefaultConfig() PluginConfig {
 		CniConfig: CniConfig{
 			NetworkPluginBinDir:       "/opt/cni/bin",
 			NetworkPluginConfDir:      "/etc/cni/net.d",
+			NetworkPluginMaxConfNum:   1,
```

Comment:
hmm.. should we be loading this from some vendor constant over in cni or go-cni? Sort of goes with the above question, where is this going to be used. Are there any test cases?

Reply:
this replaces the default .. which is by default 1 so no observable behavior change here.. just odd that we are not vendoring the default from go-cni